master/static/srv/ship_logs.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# A simple wrapper script so that we can honor DET_PYTHON_EXECUTABLE when
+# invoking the ship_logs.py.
+
+set -e
+
+if [ -z "$DET_PYTHON_EXECUTABLE" ]; then
+    export DET_PYTHON_EXECUTABLE="python3"
```

Comment:
maybe `type -P "$DET_PYTHON_EXECUTABLE"` and bail with a special exit code? (or whatever `whence -p python` is in bash).

Reply:
Can you elaborate?  I'm not really sure what you mean.

Reply:
sorry, can we check if python exists here?

Reply:
We could, but bash already emits a special exit code for command-not-found.

I could make this emit a different exit code in that case, is that what you mean?

Reply:
That is what I mean. I know it does, this just felt like a very special case of "command not found". What do you think? Not worth it?

Reply:
personally, I feel "not worth it", but it's not a bad idea.  If you want it I will do it.